bahmni_sale/models/res_config_settings.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields, api
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+   
+    sale_price_markup = fields.Boolean(string="Sale Price Markup Rule", store=True)
+
+
+    
+    def set_values(self):
+       res = super(ResConfigSettings, self).set_values()
+       self.env['ir.config_parameter'].sudo().set_param('discount.sale_price_markup', self.sale_price_markup)
```

Comment:
This parameter is not related to discount. Can we have a better key ?

Reply:
@mohan-13 , Its base _inherit module, So key name defined for discount.

Reply:
This will be refactored and moved to a new module which will hold all configurations in the upcoming PR